testData/solutions/win32.json5

```diff
@@ -20194,18 +20194,18 @@
                 ],
                 "capabilitiesTransformations": {
                     "HighContrastOn": {
-                        // (highContrast.enabled || highContrastTheme) && (highContrastTheme != "regular-contrast")
+                        // (highContrast.enabled && highContrastTheme) && (highContrastTheme != "regular-contrast")
                         "value": {
                             "transform": {
                                 "type": "fluid.transforms.binaryOp",
                                 "left": {
                                     "transform": {
                                         "type": "fluid.transforms.binaryOp",
                                         "leftPath": "http://registry\\.gpii\\.net/common/highContrast/enabled",
-                                        "left": false,
-                                        "operator": "||",
+                                        "left": true,
```

Comment:
The use of the plain "left" and "right" values appears to have also been a bit of a historical mistake - the result seems to have been something like ((fluid.isValue(highContrast.enabled) || true) && (fluid.isValue(highContrastTheme) || true) which is likely to have been something nobody wants. I recommend removing the "left" and "right" values entirely, assuming this doesn't break for yet some other reason